doc/policies.md

```diff
@@ -31,3 +31,55 @@ A release (identified by a unique semantic version) is protected against
 changes by its integrity hashes. If errors are identified post-publication, a
 release could be withdrawn, or superseded by a new one (using the appropriate
 major/minor/patch/build version changes), but not modified.
+
+## Best practices
+
+ - Avoid using `ada` as a prefix for your crate name, this will make the
+   project harder to find in a list. `ada` suffix is ok when the project is a
+   binding for an existing library (e.g. `sdlada`, `gtkada`).
+ 
+ - Split big projects in multiple crates:
+    
+    - If your project is a collection of components (like GNATcoll for
+      instance) and each component has different dependencies, you should
+      consider splitting the collection into multiple Alire crates. The
+      components can still be maintained in the same repository and use the
+      same release archive/commit (e.g.
+      [gnatcoll_sqlite](https://alire.ada.dev/crates/gnatcoll_sqlite),
+      [gnatcoll_sql](https://alire.ada.dev/crates/gnatcoll_sql),
+      [gnatcoll_postgres](https://alire.ada.dev/crates/gnatcoll_postgres)).
+    
+    - If your project is an application/executable/tool, some parts of the
+      application are potentially interesting on their own and would benefit
+      the ecosystem. For instance a parser for a standard file format.
+
+ - GPR project file clashes: to prevent issues when combining the GPR project
+   files of different crates, we recommend to follow the rules below:
+
+    - Use a project file name that matches the name of the crate (e.g.
+      `my_crate.gpr` for a crate named `my_crate`)
+
+    - Avoid using multiple GPR project files for a single crate
+
+    - Avoid using common names for GPR project files such as `shared.gpr`,
+      `common.gpr`, `config.gpr`, etc.
+
+    - Prefix GPR scenario variables with the name of your crate:
+      ```
+      Build_Mode := External ("MY_CRATE_BUILD_MODE", "release");
+      ```
+
+    - Avoid common names for GPR scenario variables such as `OS`, `TARGET`,
+      `BUILD_MODE`, `MODE`, etc.
+
+    - For library projects, do use the "standard" `LIBRARY_TYPE` external, but
+      wrap it in a crate specific external:
```

Comment:
The intention here is that the global applies by default, but that it can be overridden for a single project, right? Does it merit this extra explanation?